win32/src/async_socket_win32.c

```diff
@@ -324,18 +320,11 @@ void async_socket_destroy(ASYNC_SOCKET_HANDLE async_socket)
             }
             else if (current_state == (LONG)ASYNC_SOCKET_WIN32_STATE_CLOSED)
             {
-                if ((SOCKET)async_socket->socket_handle != INVALID_SOCKET)
-                {
-                    LogInfo("async socket destroy, closesocket(%p);", async_socket->socket_handle);
-                    /* Codes_SRS_ASYNC_SOCKET_WIN32_42_007: [ If the socket was not OPEN then async_socket_destroy shall call closesocket on the underlying socket. ]*/
-                    (void)closesocket((SOCKET)async_socket->socket_handle);

```

Comment:
This smells like a workaround done by someone (Matt) at some point. I am happy we're removing it. We should double check with Matt tomorrow whether there is some hidden thing here (I suspect that it is indeed not needed any longer, but just to make sure ... ).

Reply:
Will do.